src/DBAL/TestStatement.php

```diff
@@ -0,0 +1,206 @@
+<?php declare(strict_types=1);
+
+namespace Fazland\DoctrineExtra\DBAL;
+
+use Doctrine\DBAL\Driver\Statement;
+use Doctrine\DBAL\FetchMode;
+use Doctrine\DBAL\ParameterType;
+
+class TestStatement implements \IteratorAggregate, Statement
+{
+    /**
+     * @var mixed[]
+     */
+    private $data;
+
+    /**
+     * @var int
+     */
+    private $columnCount = 0;
```

Comment:
Could this properties be initialized in the constructor?

Reply:
Fixed